admin/app/templates/authenticated/certifications/single/info.hbs

```diff
@@ -7,8 +7,7 @@
             {{certification-info-published record=certification float=true}}État
           </h5>
           <div class="card-text">
-            {{certification-info-field value=certification.sessionId edition=false label="Session :"
-                                       linkRoute='authenticated.certifications.sessions.info'}}
+            {{certification-info-field value=certification.sessionId edition=false label="Session :" linkRoute='authenticated.certifications.sessions.info'}}
```

Comment:
Idem pour ce fichier : pourquoi ne pas transformer en angle-brackets ? et avec @ ou this ?

Reply:
Done